### DISCUSSION TITLE
Automatic generation of date metadata

### DISCUSSION THREAD

**Discussion by SnDream** [getpelican/pelican#3352] [Q&A] [answered]
Hi, I am new to using Pelican and am just learning the basics.
Since my content is managed using Git, I was wondering if there is a way to automatically generate date/modified metadata based on Git's first commit and last commit? Thanks.

**Comment by justinmayer** [comment, marked as answer]
Welcome, @SnDream. The first step is to review the documentation, which presumably you have already done. The next thing to check is the list of available plugins, which should have yielded this option: https://github.com/getpelican/pelican-plugins/tree/master/filetime_from_git

**Reply by SnDream** [reply]
Thanks, I haven't started using any of the plugins yet, I'll try it out.
